ctru-sys/src/bin/docstring-to-rustdoc.rs

```diff
@@ -33,8 +33,7 @@ fn main() -> io::Result<()> {
         .map(|v| {
             // Only modify lines with the following structure: `` #[doc ... ] ``
             if v.trim_start().starts_with("#[doc") && v.trim_end().ends_with("]") {
```

Comment:
While you're at it, might as well fix this clippy warning:
```
warning: single-character string constant used as pattern
  --> ctru-sys/src/bin/docstring-to-rustdoc.rs:35:78
   |
35 |             if v.trim_start().starts_with("#[doc") && v.trim_end().ends_with("]") {
   |                                                                              ^^^ help: try using a `char` instead: `']'`
   |
   = note: `#[warn(clippy::single_char_pattern)]` on by default
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#single_char_pattern
```

Reply:
Moved this comment to the CI PR:
https://github.com/Meziu/ctru-rs/pull/66/files#r929514001